docs/quickstart.rst

```diff
@@ -113,7 +113,7 @@ Retrieve a list of calls
     response = client.get_calls()
 
 Docs:
-`https://docs.nexmo.com/voice/voice-api/api-reference#call\_retrieve <https://docs.nexmo.com/voice/voice-api/api-reference#call_retrieve?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library>`__
+`https://docs.vonage.com/voice/voice-api/api-reference#call\_retrieve <https://docs.vonage.com/voice/voice-api/api-reference#call_retrieve?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library>`__
```

Comment:
This needs correct to point to the developer docs, this URL isn't valid. `docs.nexmo.com` is also just a redirect, so this should point to the current URL.